docs/src/tutorials/single_stage_model.md

```diff
@@ -0,0 +1,62 @@
+# [Operation Problem with `HydroPowerSimulations.jl`](@id op_problem)
+
+**Originally Contributed by:** Rodrigo Henriquez-Auba
+
+Load packages:
+
+```@repl op_problem
+using PowerSystems
+using PowerSimulations
+using HydroPowerSimulations
+using PowerSystemCaseBuilder
+using HiGHS # solver
+```
+
+## Data
+
+!!! note
+    
+    `PowerSystemCaseBuilder.jl` is a helper library that makes it easier to reproduce examples in the documentation and tutorials. Normally you would pass your local files to create the system data instead of calling the function `build_system`.
+    For more details visit [PowerSystemCaseBuilder Documentation](https://nrel-sienna.github.io/PowerSystems.jl/stable/tutorials/powersystembuilder/)
+
+```@repl op_problem
+sys = build_system(PSITestSystems, "c_sys5_hy")
+```
+
+With a single `HydroDispatch`:
+
+```@repl op_problem
+hy = only(get_components(HydroDispatch, sys))
+```
+
+## Decision Model
+
+Setting up the formulations, including hydro using `HydroDispatchRunOfRiver`
```

Comment:
Maybe break this up to clarify what is coming standard from PSI, and reiterating the hydro part is from this extension package?

Reply:
This one is defined here in HSS. The only formulation from PSI that's not defined here is `FixedOutput`